### ISSUE TITLE
Consider adding support for new gltf extension 

### ISSUE THREAD

**Issue by issacclee** [michaelgold/usdzconvert-docker#1]
Hi Michael,
Awesome project, it's nice to have another project that can do gltf to usdz conversion after the google usd_from_gltf project got abandoned.
With the introduction of Realitykit, we now have more channels to work with in terms of physically based material. Some of thoese channels can be mapped to several new gltf extensions that are added to address advanced PBR scenario. And some of those newly added extensions seems can be mapped to Realitykit one to one, such as:
- [KHR_materials_sheen](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_sheen/README.md) -> sheen(available since ios15.0 in realitykit)
- [KHR_materials_specular](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_specular/README.md) -> specular(available since ios15.0 in realitykit)
- [KHR_materials_emissive_strength](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/%5BKHR_materials_clearcoat%5D(https:/github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_emissive_strength)/README.md) ->emissiveIntensity(available since ios15.0 in realitykit)

would you consider adding support to those extensions part of this project's scope? if so, I would like to give it a go to even though my python skills are limited.

Thanks!

**Comment by michaelgold** [comment]
Hi Issaclee! 

I am currently still waiting on Apple to update their Python code in this tool: https://developer.apple.com/augmented-reality/tools/files/USDPython-pkg.zip

As soon as they do, I'll make a new release.

Hopefully, we will see some new updates this summer with the next version of iOS.
